packages/spor-react/src/theme/components/button.ts

```diff
@@ -129,14 +129,14 @@ const config = defineStyleConfig({
         backgroundColor: mode("mint", colors.whiteAlpha[300])({ colorMode }),
       },
     }),
-    ghost: () => ({
+    ghost: ({colorMode}) => ({
```

Comment:
Det er ikke noe i veien for å bare sende inn alle props her altså:

```tsx
ghost: props => ({
  color: mode("darkGrey", "white")(props),
})
```

Litt mindre skriving, og litt mindre støy